models/contacts.schema.js

```diff
@@ -0,0 +1,27 @@
+const { mongoose, Schema } = require('mongoose');
+
+const contacts = new Schema({
+  name: {
+    type: String,
+    required: [true, "Set name for contact"],
+  },
+  email: {
+    type: String,
+  },
+  phone: {
+    type: String,
+  },
+  favorite: {
+    type: Boolean,
+    default: false,
+  },
+  owner: {
+      type: Schema.Types.ObjectId,
+      ref: 'user',
+    }
```

Comment:
Zauważyłem że tego Klucza nie wykorzystujesz. On jest po to aby użytkownik miał dostęp tylko do swoich kontaktów. Czyli razem z wprowadzeniem autoryzacji powinnaś dodawać ID użytkownika który tworzy jakiś kontakt a następnie za każdym razem kiedy robisz coś z tym kontaktem to upewnić się że ten kontakt przynależy właśnie do tego użytkownika.